modules/nlu/src/backend/application/typings.ts

```diff
@@ -22,8 +21,8 @@ export interface BotDefinition {
 export type ProgressCallback = (p: number) => Promise<void>
 
 export interface Trainable {
-  train(language: string, progressCallback: ProgressCallback): Promise<NLU.ModelId>
-  load(modelId: NLU.ModelId): Promise<void>
+  train(language: string, progressCallback: ProgressCallback): Promise<string>
+  setModel(language: string, modelId: string): void
```

Comment:
why the change from load to setModel, is STAN server loading the model and the client is just setting it?

Reply:
the client doesnt load the model in engine anymore, its Stan responsability. Stan only lazy-loads models now when they are required at predict time.